src/Tooltip/Content.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react'
+import css from 'styled-jsx/css'
+import cx from 'classnames'
+
+import { invertSide } from './helpers'
+
+export const Content = React.forwardRef(
+    ({ side, children, position, withArrow, adjustment }, ref) => (
+        <div
+            ref={ref}
+            style={position}
+            className={cx('container', {
+                'with-arrow': withArrow,
+                [`with-arrow-${invertSide(side)}`]: withArrow,
+            })}
+        >
+            <div className="content">{children}</div>
+
+            <style jsx>{`
+                .container {
+                    background: white;
+                    box-shadow: 0 0 3px rgba(0, 0, 0, 0.6);
+                    position: absolute;
+                }
+
+                .content {
+                    position: relative;
+                    z-index: 2;
```

Comment:
Avoid `z-index` in general.

`z-index` of `2` is illegal since it is within the application stack.

Reply:
As discussed, z-index inside elements with z-index are relative inside but to not effect the outside z-index and other elements that have the same z-index as the outer one